06_gpu_and_ml/text-to-pokemon/text_to_pokemon/main.py

```diff
@@ -112,11 +60,15 @@ def image_to_byte_array(image) -> bytes:
         return buf.getvalue()
 
 
+@stub.cls(gpu="A10G", shared_volumes={config.CACHE_DIR: volume}, keep_warm=1)
 class Model:
     def __enter__(self):
-        self.pipe = load_stable_diffusion_pokemon_model().to("cuda")
+        import threading
+        if not pokemon_naming.rnn_names_output_path.exists():
```

Comment:
not thread safe at scale but most of these examples aren't either, going to just defer to precedent